x-pack/plugins/security/server/lib/role_schema.js

```diff
@@ -19,6 +19,7 @@ export const roleSchema = {
     query: Joi.string().allow('')
   }),
   run_as: Joi.array().items(Joi.string()),
+  applications: Joi.array(),
```

Comment:
I don't think we're using this file anymore: it defines ES's role schema, which differs from the one we use in the public Kibana roles API.

We could consider replacing this with the inline schema defined in the `put.js` route handler, but I realize that's dynamically built because of an earlier request I made 😄. No need to do anything about that, but we should at least delete this file.